src/lib/utils.js

```diff
@@ -68,18 +66,16 @@ function addFileToFormData(uri, name, contentType) {
 }
 
 function replaceStreamObjects(obj) {
-  let cloned = obj;
-  if (Array.isArray(obj)) {
-    cloned = obj.map((v) => replaceStreamObjects(v));
-  } else if (isPlainObject(obj)) {
-    cloned = {};
+  if (Array.isArray(obj)) return obj.map((v) => replaceStreamObjects(v));
+  if (Object.prototype.toString.call(obj) === '[object Object]') {
+    const cloned = {};
     Object.keys(obj).forEach((k) => {
       cloned[k] = replaceStreamObjects(obj[k]);
     });
-  } else if (isObject(obj) && obj.ref && typeof obj.ref === 'function') {
```

Comment:
it seems it was handling null which we don't now.

Reply:
yea test failed because of that, pushed a fix